is-within-range/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <script>
+    function isWithinRange(number, rangeObject) {
+      if (number <= rangeObject.max && number >= rangeObject.min) {
+        return true
+      } else {
+        return false
+      }
+    }
+
+    // function taking in 2 parameters, number and object with max and min numbers
+    // create if else statement, if number matches two conditions return true 
```

Comment:
What two conditions? Make sure you elaborate on this sort of thing. remember, whoever is reading your pseudocode should have a full understanding of the problem and how to solve it based on reading the instructions.